op/upgrade_v2.go

```diff
@@ -7,31 +7,29 @@ import (
 	"github.com/cybozu-go/well"
 )
 
-type upgradeOp struct {
-	current string
-	nodes   []*cke.Node
+type v2UpgradeOp struct {
+	nodes     []*cke.Node
+	completed bool
 }
 
-// UpgradeOp returns an Operator to upgrade cluster configuration.
-func UpgradeOp(current string, nodes []*cke.Node) cke.Operator {
-	return &upgradeOp{current: current, nodes: nodes}
+// V2UpgradeOp returns an Operator to upgrade cluster configuration.
+func V2UpgradeOp(nodes []*cke.Node) cke.Operator {
+	return &v2UpgradeOp{nodes: nodes}
 }
 
-func (u *upgradeOp) Name() string {
-	return "upgrade"
+func (u *v2UpgradeOp) Name() string {
+	return "upgrade-v2"
 }
 
-func (u *upgradeOp) NextCommand() cke.Commander {
-	switch u.current {
-	case "1":
-		u.current = "2"
-		return UpgradeToVersion2Command(u.nodes)
-	default:
+func (u *v2UpgradeOp) NextCommand() cke.Commander {
+	if u.completed {
 		return nil
 	}
+	u.completed = true
+	return UpgradeToVersion2Command(u.nodes)
 }
```

Comment:
I think `upgradeOp` should be kept intact.

Just adding `case "2"` to `NextCommand()` is enough, isn't it?